src/main/scala/mesosphere/marathon/api/v2/validation/PodsValidation.scala

```diff
@@ -189,6 +188,7 @@ trait PodsValidation {
 
   def containerValidator(pod: Pod, enabledFeatures: Set[String], mesosMasterVersion: SemanticVersion): Validator[PodContainer] =
     validator[PodContainer] { container =>
+      container.name should matchRegexFully("^[a-z0-9]([-a-z0-9]*[a-z0-9])?$")
```

Comment:
any reason that we're not using `validName`? https://github.com/mesosphere/marathon/blob/releases/1.4/src/main/scala/mesosphere/marathon/api/v2/validation/PodsValidation.scala#L28